src/lib/mat-exports.ts

```diff
@@ -0,0 +1,526 @@
+export {MdRadioGroup as MatRadioGroup};
+export {MdRadioGroupBase as MatRadioGroupBase};
+export {MdRadioModule as MatRadioModule};
+export {MdRaisedButtonCssMatStyler as MatRaisedButtonCssMatStyler};
+export {MdRipple as MatRipple};
+export {MdRippleModule as MatRippleModule};
+export {MdRow as MatRow};
+export {MdSelect as MatSelect};
+export {MdSelectBase as MatSelectBase};
+export {MdSelectChange as MatSelectChange};
+export {MdSelectionList as MatSelectionList};
+export {MdSelectionListBase as MatSelectionListBase};
+export {MdSelectionListOptionEvent as MatSelectionListOptionEvent};
+export {MdSelectModule as MatSelectModule};
+export {MdSelectTrigger as MatSelectTrigger};
+export {MdSidenav as MatSidenav};
+export {MdSidenavContainer as MatSidenavContainer};
+export {MdSidenavModule as MatSidenavModule};
+export {MdSlider as MatSlider};
+export {MdSliderBase as MatSliderBase};
+export {MdSliderChange as MatSliderChange};
+export {MdSliderModule as MatSliderModule};
+export {MdSlideToggle as MatSlideToggle};
+export {MdSlideToggleBase as MatSlideToggleBase};
+export {MdSlideToggleChange as MatSlideToggleChange};
+export {MdSlideToggleModule as MatSlideToggleModule};
+export {MdSnackBar as MatSnackBar};
+export {MdSnackBarConfig as MatSnackBarConfig};
+export {MdSnackBarContainer as MatSnackBarContainer};
+export {MdSnackBarModule as MatSnackBarModule};
+export {MdSnackBarRef as MatSnackBarRef};
+export {MdSort as MatSort};
+export {MdSortable as MatSortable};
+export {MdSortHeader as MatSortHeader};
+export {MdSortHeaderIntl as MatSortHeaderIntl};
+export {MdSortModule as MatSortModule};
+export {MdSpinner as MatSpinner};
+export {MdTab as MatTab};
+export {MdTabBody as MatTabBody};
+export {MdTabBodyOriginState as MatTabBodyOriginState};
+export {MdTabBodyPositionState as MatTabBodyPositionState};
+export {MdTabChangeEvent as MatTabChangeEvent};
+export {MdTabGroup as MatTabGroup};
+export {MdTabGroupBase as MatTabGroupBase};
+export {MdTabHeader as MatTabHeader};
+export {MdTabHeaderPosition as MatTabHeaderPosition};
+export {MdTabLabel as MatTabLabel};
+export {MdTabLabelWrapper as MatTabLabelWrapper};
+export {MdTable as MatTable};
+export {MdTableModule as MatTableModule};
+export {MdTabLink as MatTabLink};
+export {MdTabNav as MatTabNav};
+export {MdTabsModule as MatTabsModule};
+export {MdTextareaAutosize as MatTextareaAutosize};
+export {MdToolbar as MatToolbar};
+export {MdToolbarBase as MatToolbarBase};
+export {MdToolbarModule as MatToolbarModule};
+export {MdToolbarRow as MatToolbarRow};
+export {MdTooltip as MatTooltip};
+export {MdTooltipModule as MatTooltipModule};
+export {MdYearView as MatYearView};
+export {MdFormFieldModule as MatFormFieldModule};
+export {MdError as MatError};
+export {MdFormField as MatFormField};
+export {MdFormFieldControl as MatFormFieldControl};
+export {MdHint as MatHint};
+export {MdPlaceholder as MatPlaceholder};
+export {MdPrefix as MatPrefix};
+export {MdSuffix as MatSuffix};
```

Comment:
I really hope this wasn't done manually :P

Reply:
Nope, Andrew made a script that generated it. We've had this inside Google for a while now but needed 4.4 to come out to make it work externally, so I just copy-pasted the internal file.